guides/process/scheduling/setting_up_projects.md

```diff
@@ -0,0 +1,127 @@
+# Setting up Projects
+
+We use [Harvest](https://madetech.harvestapp.com/) and [Forecast](https://forecastapp.com/967648/schedule/projects) to track and project time spent on both internal and customer-facing projects at Made Tech.
+
+It is important that we set up projects consistently, as accurate time tracking underpins a number of important things:
+
+- We almost always invoice customers on a Time and Materials (T&M) basis, which means timesheets for customer engagements are converted directly in to invoices
+
+- There are various transactions that happen between markets (e.g. for a London person spending time working on a delivery in Manchester) that impact revenue and cost in both markets
+
+- Service Areas have a set budget that they're working to
+
+## Workflow
+
+There are a number of steps to setting up a new project. The remainder of thise guide describes the context behind the decisions you'll need to make when setting up a new project.
+
+1. Create project in Harvest
+2. Assign people and rates to project in Harvest
+3. Import project to Forecast
+4. Forecast people against project in Harvest
+5. Ensure the team know what to timesheet against
+
+Please be encouraged to ask questions in `#ops-scheduling` if you need help. 
```

Comment:
Can we add that this is a Slack channel, this wasn't clear for one of the new joiners not familiar with Slack and where to find this.